zerver/tests/test_new_users.py

```diff
@@ -26,7 +27,8 @@ def test_send_login_emails_if_send_login_email_setting_is_true(self):
 
```

Comment:
I've modified the test so it spoofs a user agent of Firefox on Windows and checks that the email says that as well